ontologies/WIP/sierratransformable

```diff
@@ -277,27 +275,27 @@
    *{
    *  "subjects": [
    *    {
-   *      "label": "Tuberculosis - prevention & control.",
+   *      "label": "Tuberculosis -- prevention & control -- Paris -- 1850 -- bibliography.",
```

Comment:
I do wonder if we want to agree a scheme for adding a label that makes is fairly display logic agnostic (i.e. make this data a bit more obviously a delimiter separated string).

Reply:
Yes this is something I meant to bring up. Though the librarians have suggested double hyphens it does look odd to me.

Perhaps one we need to speak to Experience team about?

Reply:
Paging @jamesgorrie!

Reply:
@kenoir @silveroliver You might need to run me through what this text is trying to say?
As it stands I can't really infer anything.

Reply:
Down with double hyphens! I will be very surprised if we actually need to use them in the UI; we're more likely to normalise to single hyphens if needed.

Reply:
So the question is ... is the label at all useful to anyone? Is there a way to make it more useful to consumers?

Reply:
We don't know that yet because we haven't done any subject-string testing.

Reply:
So maybe we just leave it as it is and reindex later based on testing?